### DISCUSSION TITLE
How to follow instructions re commit/stash local changes after failed sync/update?

### DISCUSSION THREAD

**Discussion by thyttan** [LazyVim/LazyVim#368] [Q&A] [answered]
Since a couple of days, when I sync or update I get:

>   Failed (2)
>     ● LuaSnip 52.19ms   cmp_luasnip
>         error: Your local changes to the following files would be overwritten by checkout:
>         	DOC.md
>         	doc/luasnip.txt
>         Please commit your changes or stash them before you switch branches.
>         Aborting
>     ● mini.ai 44.8ms  VeryLazy
>         error: Your local changes to the following files would be overwritten by checkout:
>         	doc/mini-ai.txt
>         	lua/mini/ai.lua
>         Please commit your changes or stash them before you switch branches.
>         Aborting

I don't remember manually altering the files mentioned, and I don't find them when searching in Nautilus file manager. 

I did uncomment 

>     -- { import = "lazyvim.plugins.extras.lang.json" },
>     -- { import = "lazyvim.plugins.extras.ui.mini-animate" },

in lazy.lua at some point but have since commented them out again.

My fork of LazyVim starter which is synced with my setup is [here](https://github.com/thyttan/LazyVim-starter/tree/personalized).

How do I follow the instructions in the output to the "Failed"-field?

Alternatively, how do I force the update and overwrite the files?

**Comment by folke** [comment, marked as answer]
Just remove the plugins in the LAzy ui (with `x`). and install them again with `I`

And **never** change any files in the lazy plugins dir

**Reply by thyttan** [reply]
>Just remove the plugins in the LAzy ui (with x). and install them again with I

Thanks, that worked!

>And never change any files in the lazy plugins dir

Ok, I'm not sure I did though.

**Reply by folke** [reply]
I'm sure you didn't, since those files were changed...
